northstar/src/api/model.rs

```diff
@@ -127,9 +142,16 @@ pub enum Response {
     Containers(Vec<ContainerData>),
     Repositories(HashSet<RepositoryId>),
     Mount(Vec<MountResult>),
+    ContainerStats(ContainerStats),
     Err(Error),
 }
 
+#[derive(new, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct ContainerStats {
+    pub container: Container,
+    pub stats: HashMap<String, serde_json::Value>,
```

Comment:
Maybe use `crate::runtime::stats::ContainerStats` or some alias for the type instead?

Reply:
Done.